app.rb

```diff
@@ -1,9 +1,54 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/message'
+
+# add check when search returns empty
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+  configure :development do
+    register Sinatra::Reloader
+  end 
+
+  enable :sessions
+
+  get '/' do
+    @messages = Message.all
+    erb :index
+  end
+
+  get '/new' do
+    erb :new
+  end 
+
+  get '/search' do
+    erb :search
+  end 
+
+  get '/search_result' do
```

Comment:
Do you _have_ to have a route specifically for this?  Could you combine it with `/`?